content/professional/books.md

```diff
@@ -11,23 +11,26 @@ We like these Books
   <div class="column is-one-third">
      {{% card 
          title="Discovery"
-         image="/img/book-discovery.png"
+         image="/img/book-discovery.png" 
+         link="https://www.goodreads.com/book/show/36126762-the-bdd-books"
```

Comment:
I like having individual links per book, rather than go to google home page. Not sure it should go to GoodReads though. @aslakhellesoy @mattwynne @sebrose - do you have an opinion on this?

Reply:
Ya, I felt the same. But then the reason for choosing GoodReads is that it provides genuine user reviews, multiple vendors and formats (Physical, digital & audio) for the user to choose from. Let's hear from others too.

Reply:
From an author's point of view, linking to the Pragmatic Programmers' page for the books that are published through them means we get a better royalty payment than if you link to Amazon. Otherwise, just do what seems most useful to the reader.